app/src/main/res/layout/dashboard_tools_spotlight_list_item.xml

```diff
@@ -0,0 +1,105 @@
+        <import type="android.text.TextUtils"/>
+        <import type="android.view.View" />
+
+        <variable name="callbacks" type="ObservableField&lt;org.cru.godtools.ui.tools.ToolsAdapterCallbacks&gt;" />
+        <variable name="tool" type="org.cru.godtools.model.Tool" />
+        <variable name="banner" type="LiveData&lt;org.cru.godtools.model.Attachment&gt;" />
+        <variable name="downloadProgress" type="LiveData&lt;org.cru.godtools.download.manager.DownloadProgress&gt;" />
+        <variable name="primaryTranslation" type="LiveData&lt;org.cru.godtools.model.Translation&gt;" />
+        <variable name="parallelTranslation" type="LiveData&lt;org.cru.godtools.model.Translation&gt;" />
+        <variable name="parallelLanguage" type="LiveData&lt;org.cru.godtools.model.Language&gt;" />
+    </data>
+
+    <androidx.cardview.widget.CardView
+        android:layout_width="200dp"
+        android:layout_height="240dp"
+        android:layout_margin="8dp"
+        android:elevation="@dimen/card_elevation"
+        android:minHeight="240dp"
+        android:onClick="@{() ->callbacks.onToolInfo(tool.code)}"
+        app:cardCornerRadius="@dimen/card_corner_radius"
+        tools:layout_height="240dp">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layoutDirection="@{primaryTranslation.languageCode != null ? TextUtils.getLayoutDirectionFromLocale(primaryTranslation.languageCode) : View.LAYOUT_DIRECTION_INHERIT}">
+
+
+            <org.cru.godtools.base.ui.view.DaggerPicassoImageView
+                android:id="@+id/toolsImage"
+                style="@style/Widget.GodTools.Tools.Card.Banner"
+                android:layout_width="match_parent"
+                android:layout_height="161dp"
+                android:scaleType="centerCrop"
+                android:src="@{banner}"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                tools:src="@tools:sample/backgrounds/scenic" />
+
+            <ProgressBar
+                style="@style/Widget.GodTools.Tools.Download.ProgressBar"
+                android:layout_width="match_parent"
+                android:layout_height="10dp"
+                android:progress="@{downloadProgress}"
+                android:visibility="gone"
+                app:layout_constraintBottom_toBottomOf="@id/toolsImage"
+                tools:max="12"
+                tools:progress="5"
+                tools:visibility="visible" />
+
+            <com.google.android.material.imageview.ShapeableImageView
+                style="@style/Widget.GodTools.Tools.Card.Action.Favorite"
+                android:layout_marginTop="11dp"
+                android:layout_marginEnd="11dp"
+                android:elevation="@dimen/card_elevation"
```

Comment:
the elevation is centrally managed by the `Widget.GodTools.Tools.Card.Action.Favorite` style